### ISSUE TITLE
Can't archive in Xcode 7.3 (7D175)

### ISSUE THREAD

**Issue by thomascalmon** [sgr-ksmt/PDFGenerator#21]
Hello,

I'm trying to build my app that uses PDFGenerator and I'm getting the error bellow:

```
SwiftCodeGeneration normal arm64 /Users/thomas/Library/Developer/Xcode/DerivedData/DanFlow_5000-hkzfkytwkxmfncfdsbfhkketzfus/Build/Intermediates/ArchiveIntermediates/DanFlow 5000/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/PDFGenerator.build/Objects-normal/arm64/PDFGenerator.bc
    cd /Volumes/DADOS/Dev/iOS/DanFlow 5000/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c -primary-file "/Users/thomas/Library/Developer/Xcode/DerivedData/DanFlow_5000-hkzfkytwkxmfncfdsbfhkketzfus/Build/Intermediates/ArchiveIntermediates/DanFlow 5000/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/PDFGenerator.build/Objects-normal/arm64/PDFGenerator.bc" -target arm64-apple-ios8.0 -Xllvm -aarch64-use-tbi -O -module-name PDFGenerator -o "/Users/thomas/Library/Developer/Xcode/DerivedData/DanFlow_5000-hkzfkytwkxmfncfdsbfhkketzfus/Build/Intermediates/ArchiveIntermediates/DanFlow 5000/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/PDFGenerator.build/Objects-normal/arm64/PDFGenerator.o" -embed-bitcode -disable-llvm-optzns

0  swift                    0x000000010aaa14eb llvm::sys::PrintStackTrace(llvm::raw_ostream&) + 43
1  swift                    0x000000010aaa07d6 llvm::sys::RunSignalHandlers() + 70
2  swift                    0x000000010aaa1b4f SignalHandler(int) + 287
3  libsystem_platform.dylib 0x00007fff9298352a _sigtramp + 26
4  libsystem_platform.dylib 0x00007fd808461450 _sigtramp + 1974329152
5  swift                    0x000000010a14d6fa llvm::DwarfCompileUnit::applyVariableAttributes(llvm::DbgVariable const&, llvm::DIE&) + 74
6  swift                    0x000000010a151648 llvm::DwarfDebug::finishVariableDefinitions() + 376
7  swift                    0x000000010a1519ae llvm::DwarfDebug::finalizeModuleInfo() + 126
8  swift                    0x000000010a151cf7 llvm::DwarfDebug::endModule() + 39
9  swift                    0x000000010a133ef7 llvm::AsmPrinter::doFinalization(llvm::Module&) + 839
10 swift                    0x000000010a8d4415 llvm::FPPassManager::doFinalization(llvm::Module&) + 53
11 swift                    0x000000010a8d48f1 llvm::legacy::PassManagerImpl::run(llvm::Module&) + 1185
12 swift                    0x000000010891444b performLLVM(swift::IRGenOptions&, swift::DiagnosticEngine&, llvm::sys::SmartMutex<false>*, llvm::Module*, llvm::TargetMachine*, llvm::StringRef) + 1067
13 swift                    0x0000000108914571 swift::performLLVM(swift::IRGenOptions&, swift::ASTContext&, llvm::Module*) + 145
14 swift                    0x00000001087f317a performCompile(swift::CompilerInstance&, swift::CompilerInvocation&, llvm::ArrayRef<char const*>, int&) + 506
15 swift                    0x00000001087f241d frontend_main(llvm::ArrayRef<char const*>, char const*, void*) + 2781
16 swift                    0x00000001087ede3c main + 1932
17 libdyld.dylib            0x00007fff928605ad start + 1
18 libdyld.dylib            0x0000000000000010 start + 1836710500
Stack dump:
0.	Program arguments: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c -primary-file /Users/thomas/Library/Developer/Xcode/DerivedData/DanFlow_5000-hkzfkytwkxmfncfdsbfhkketzfus/Build/Intermediates/ArchiveIntermediates/DanFlow 5000/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/PDFGenerator.build/Objects-normal/arm64/PDFGenerator.bc -target arm64-apple-ios8.0 -Xllvm -aarch64-use-tbi -O -module-name PDFGenerator -o /Users/thomas/Library/Developer/Xcode/DerivedData/DanFlow_5000-hkzfkytwkxmfncfdsbfhkketzfus/Build/Intermediates/ArchiveIntermediates/DanFlow 5000/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/PDFGenerator.build/Objects-normal/arm64/PDFGenerator.o -embed-bitcode -disable-llvm-optzns
```

It was OK before I updated XCode. :(
Thank you.

**Comment by benbahrenburg** [comment]
Running into the same problem. Looks as though Swift 2.2 is having an issue with this switch statement https://github.com/sgr-ksmt/PDFGenerator/blob/master/PDFGenerator/PDFGenerator.swift#L298

**Comment by Lejdborg** [comment]
The problem seems to specifically lie in this snippet:

```swift
case .WhitePage(let size):
    try autoreleasepool {
        let view = UIView(frame: CGRect(origin: .zero, size: size))
        view.backgroundColor = .whiteColor()
        try view.renderPDFPage()
    }
```

I temporarily removed that part and got my build working.

**Comment by thomascalmon** [comment]
Thank you, guys.
I got build working too.

**Comment by johnslay** [comment]
Had the same problem. I just completely removed the switch statement. PDF seems to still render fine.

**Comment by thomascalmon** [comment]
In the end, I preferred remove PDFGenerator from my project! I didnt make PDF render without switch statement. :(

**Comment by sgr-ksmt** [comment]
Sorry for replying late. :cry: 
I fixed and update to [1.1.4](https://github.com/sgr-ksmt/PDFGenerator/releases/tag/1.1.4).

**Comment by thomascalmon** [comment]
Don't be sorry, I'm sure You did your best.
Thanks for replying and fixing the bug.

Cheers!
